README.md

```diff
@@ -31,8 +31,19 @@ $ export SECRETLY_NAMESPACE=foo/bar
 $ secretly env | grep BAZ
 BAZ=mysecretpassword
 ```
+In addition, it's possible to specify multiple namespaces by separating them with commas in the `SECRETLY_NAMESPACE` variable.
```

Comment:
I think it would be helpful to state that order matters: if there is a variable name conflict, namespaces further in the list will overwrite namespaces earlier in the list.